components/SiteNavbar.vue

```diff
@@ -80,25 +93,69 @@
         </button>
       </div>
     </div>
+
+    <transition
+      enter-active-class="transition-opacity duration-200"
+      leave-active-class="transition-opacity duration-200"
+      enter-class="opacity-0"
+      enter-to-class="opacity-100"
+      leave-class="opacity-100"
+      leave-to-class="opacity-0"
+    >
+      <BaseModal v-show="showModal" @close="showModal = false">
+        <LogInCard
+          v-show="showLogin"
+          class="shadow-xl"
+          :modal="true"
+          @close="showModal = false"
+          @signUpClick="showLogin = false"
+        />
+        <SignUpCard
+          v-show="!showLogin"
+          class="shadow-xl"
+          :modal="true"
+          @close="showModal = false"
+          @logInClick="showLogin = true"
+        />
+      </BaseModal>
+    </transition>
   </div>
 </template>
 
 <script lang="ts">
 import { defineComponent, ref } from '@nuxtjs/composition-api'
-import { useBreakpoints } from "@vueuse/core"
-import { screens } from "@/breakpoints";
+import { useBreakpoints } from '@vueuse/core'
+import { screens } from '@/breakpoints'
 
-const breakpoints = useBreakpoints(screens);
+const breakpoints = useBreakpoints(screens)
 
 export default defineComponent({
   setup() {
-    const lg = breakpoints.greater("lg");
-    const mobileNavIsActive = ref(false);
+    const lg = breakpoints.greater('lg')
+    const mobileNavIsActive = ref(false)
 
     return {
       lg,
       mobileNavIsActive,
     }
   },
+
+  data() {
+    return {
+      showModal: false,
+      showLogin: false,
+    }
+  },
```

Comment:
I think you can just return these in the `return` from the `setup()` method

Reply:
Forgot about that. Im still new to the composition API